app/src/main/res/layout/fragment_summary_product.xml

```diff
@@ -136,6 +136,39 @@
 
             </RelativeLayout>
 
+            <Button
+                android:id="@+id/btn_eating"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="8dp"
+                android:padding="8dp"
+                android:background="@drawable/button_grey_border"
+                android:drawableLeft="@drawable/ic_add_24dp"
+                android:text="@string/txt_im_eating_this" />
+
+            <Button
+                android:id="@+id/btn_eating_ticked"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="8dp"
+                android:background="@drawable/rounded_button"
+                android:drawableLeft="@drawable/ic_check_white_24dp"
+                android:padding="8dp"
+                android:text="@string/txt_im_eating_this"
+                android:textColor="@color/white"
+                android:visibility="gone" />
+
+            <Button
+                android:drawableLeft="@drawable/ic_compare_arrows_black_18dp"
```

Comment:
There is an issue as the button with the id "@+id/compare_product_button" is defined twice in this file